src/main/kotlin/ch/hevs/cloudio/cloud/apiutils/EndpointManagementUtil.kt

```diff
@@ -116,10 +120,22 @@ object EndpointManagementUtil {
         else {
             //send message to amq.topic queue
 
-            // TODO: Detect actual serialization format from endpoint data model.
+            val endpointUUID = attributeSetRequest.attributeTopic.split("/")[0]
+            val endpoint = endpointEntityRepository.findById(UUID.fromString(endpointUUID)).get()
+            val serializedAttribute =
```

Comment:
Check out the solution I did in the PostgreSQL branch:
```kotlin
fun Collection<SerializationFormat>.fromIdentifiers(identifiers: Collection<String>) = this.firstOrNull {
    (if (identifiers.isEmpty()) listOf("JSON") else identifiers).contains(it.identifier())
}
```

Then we can just:

```kotlin
val serializationFormat = serializationFormats.fromIdentifiers(endpoint.dataModel.supportedFormats)
                        ?: throw CloudioHttpExceptions.InternalServerError("Endpoint does not support any serialization format.")
                rabbitTemplate.convertAndSend("amq.topic",
                        modelIdentifier.toAmqpTopic(),
                        serializationFormat.serializeAttribute(Attribute(
                                data.constraint,
                                data.type,
                                System.currentTimeMillis().toDouble() / 1000,
                                typedValue)))
```

Reply:
Oh nice, I'll do it like that